pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -940,6 +954,23 @@ private static Set<String> getSegmentPartitionedColumns(@Nullable TableConfig ta
     return segmentPartitionedColumns;
   }
 
+  /**
+   * Retrieve multivalued columns for a table.
+   * From the table Schema , we get the multi valued columns of dimension fields.
+   *
+   * @param tableSchema
+   * @param columnName
+   * @return multivalued columns of the table .
+   */
+  private static boolean checkIfColumnIsMultiValued(@Nonnull Schema tableSchema, @Nonnull String columnName) {
```

Comment:
(minor, convention) We don't usually annotate `Nonnull` and assume non-null unless annotated with `Nullable`. We don't annotate both of them because it is easy to mix them
```suggestion
  private static boolean isMultiValueColumn(Schema schema, String columnName) {
```